README.md

```diff
@@ -1,198 +1,172 @@
-<!--
-  README.md (Partial Snippet)
-  ===========================
-  This is a work-in-progress README for YourBench. 
-  There is more coming soon—stay tuned!
--->
-
 <div align="center">
 
-<!-- Replace the paths below with your actual SVG logo paths or PNGs 
-     Make sure these files exist in docs/assets or an accessible directory -->
-
 <picture>
   <source media="(prefers-color-scheme: dark)" srcset="docs/assets/yourbench_banner_dark_mode.svg">
   <source media="(prefers-color-scheme: light)" srcset="docs/assets/yourbench_banner_light_mode.svg">
-  <img alt="YourBench Logo" src="docs/assets/yourbench_banner_light_mode.svg" width="50%" height="50%">
+  <img alt="YourBench Logo" src="docs/assets/yourbench_banner_light_mode.svg" width="50%">
 </picture>
 
 <h2>YourBench: A Dynamic Benchmark Generation Framework</h2>
 
+<a href="https://github.com/huggingface/yourbench/stargazers">
+  <img src="https://img.shields.io/github/stars/huggingface/yourbench?style=social" alt="GitHub Repo stars">
+</a>
+
 <p>
   <strong>
-    [<a href="https://github.com/huggingface/yourbench">GitHub</a>] 
-    &middot; 
-    [<a href="https://huggingface.co/datasets/sumuks/tempora">Dataset</a>] 
-    &middot; 
-    [<a href="https://github.com/huggingface/yourbench/tree/main/docs">Documentation</a>]
-    &middot;
+    [<a href="https://github.com/huggingface/yourbench">GitHub</a>] · 
+    [<a href="https://huggingface.co/datasets/sumuks/tempora">Dataset</a>] · 
+    [<a href="https://github.com/huggingface/yourbench/tree/main/docs">Documentation</a>] · 
     [<a href="https://arxiv.org/abs/2504.01833">Paper</a>]
   </strong>
 </p>
 
-<!-- Example badges -->
-<a href="https://github.com/huggingface/yourbench/stargazers">
-  <img src="https://img.shields.io/github/stars/huggingface/yourbench?style=social" alt="GitHub Repo stars">
+<a href="https://youtu.be/mhszO6kZSbI" target="_blank">
+  <img src="https://img.youtube.com/vi/mhszO6kZSbI/maxresdefault.jpg" alt="YourBench Demo Video" width="600" style="border-radius: 10px; box-shadow: 0 4px 8px rgba(0,0,0,0.1);" />
+  <br/>
+  <img src="https://img.shields.io/badge/Watch%20Demo-YouTube-red?style=for-the-badge&logo=youtube" alt="Watch Demo on YouTube"/>
+  <br/>
+  <em>Watch our 3-minute demo of the YourBench pipeline</em>
```

Comment:
Did you change the demo video? Iirc, it was not explicit enough at the time, so if unchanged I would put it at the very bottom of the readme